indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -104,6 +105,10 @@ public final static InputRow parseInputRow(Object value, InputRowParser parser)
     if(parser instanceof StringInputRowParser && value instanceof Text) {
       //Note: This is to ensure backward compatibility with 0.7.0 and before
       return ((StringInputRowParser) parser).parse(value.toString());
+    } else if(parser instanceof StringInputRowParser && value instanceof BytesWritable) {
```

Comment:
@himanshug Are you concerned that using the BytesWritable class in a Parser is going to make deserialization of the configs trigger loading of Hadoop classes, and that would be bad? If so then yeah, maybe we should put the special handling in the job itself.

But- doesn't that also mean that it's not really possible for folks to write Parsers for any sort of Hadoop object? That's kind of a bummer if so.